app/services/compute_resource_key_pairs/ec2.rb

```diff
@@ -0,0 +1,4 @@
+module ComputeResourceKeyPairs
+  class EC2 < Base
```

Comment:
I wish we had generic KeyPair object which would use adapter/mapper for each CR type. Adding these children objects leads to inconsistent API among CR types and many `ifs` in view layer. If we don't need it now, I'd suggest keeping it in a single, shared class. If there will be some changes required later, I'd like to adopt the adapter pattern.